### ISSUE TITLE
Support RCTC folder structure

### ISSUE THREAD

**Issue by TWonderchild** [OpenRCT2/OpenRCT2#6570]
**OS:** Win 10
**Version:** any

Open openRCT2 cannot find g1.dat because its in a diffrent folder in the classic edition it is in "..\RollerCoaster Tycoon Classic\Assets\"

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Multiplayer?

**Steps to reproduce:**
1. Install
2. Pick RCTC folder

**Dump file**

**Screenshots / Video:**

**Save game:**

**Comment by Gymnasiast** [comment]
It is on our list. In the meantime, you could create an artificial directory structure and copy the relevant files.

**Comment by TWonderchild** [comment]
I made a system link for the Data => Assets folder but I don't know what else I need to change since im new to openRTC2. The game is now starting but there are no scenarios.

**Comment by janisozaur** [comment]
The scenarios from classic may not be supported yet.

**Comment by Gymnasiast** [comment]
That's right. Currently only g1.dat (graphics), objects and track designs from RCTC are supported.

RCTC scenarios are in a completely different format unfortunately. So those are not yet supported.
And music is in .ogg format rather than .wav, which will need some work, but shouldn't too difficult.

**Comment by Gymnasiast** [comment]
Implemented in https://github.com/OpenRCT2/OpenRCT2/commit/a5a6d3a66c467a2834ce8efdb3ffa4d0546f8a2a . OGG and .sea support were added earlier.

**Comment by karst** [comment]
OpenRCT2 now fully supports RCTC without having to do any fniffnaf. Just select the folder where it's installed when loading the game for the first time and it'll work

**Comment by ThiaJay** [comment]
> OpenRCT2 now fully supports RCTC without having to do any fniffnaf. Just select the folder where it's installed when loading the game for the first time and it'll work

The reason I posted is because I have installed RCTC today through Steam, ran it and then installed OpenRCT2. When I tried to run OpenRCT2 and pointed it to the Steam directory C:\Program Files (x86)\Steam\steamapps\common\RollerCoaster Tycoon Classic it says it cannot find /Data/g1.dat.

I downloaded OpenRCT2 0.4.1 from https://openrct2.org/downloads/releases/latest.
